src/NServiceBus.Core/Transports/Msmq/MsmqQueueCreator.cs

```diff
@@ -92,6 +96,7 @@ static void CreateQueue(string queuePath, string account, bool transactional)
             }
 
             Logger.DebugFormat("Created queue, path: [{0}], account: [{1}], transactional: [{2}]", queuePath, account, transactional);
+            Logger.WarnFormat("Queue created with [{0}] and [{1}] permissions. Consider changing those, as required", MsmqConstants.LocalEveryoneGroupName, MsmqConstants.LocalAnonymousLogonName);
```

Comment:
This seems weird to me, so we are saying that `Everyone` is probably not the right ACL, but we are still adding it by default, so there is no way for the user to avoid this warning!